tutorials/12-gaussian-process-latent-variable-model/12_gaussian-process-latent-variable-model.jmd

```diff
@@ -0,0 +1,305 @@
+---
+title: Gaussian Process Latent Variable Model
+permalink: /:collection/:name/
+redirect_from: tutorials/12-gaussian-process-latent-variable-model/
+---
+
+# Gaussian Process Latent Variable Model
+
+In a previous tutorial, we have discussed latent variable models, in particular probabilistic principal component analysis (pPCA).
+Here, we show how we can extend the mapping provided by pPCA to non-linear mappings between input and output.
+For more details about the Gaussian Process Latent Variable Model (GPLVM),
+we refer the reader to the [original publication](https://jmlr.org/papers/v6/lawrence05a.html) and a [further extension](http://proceedings.mlr.press/v9/titsias10a/titsias10a.pdf).
+
+In short, the GPVLM is a dimensionality reduction technique that allows us to embed a high-dimensional dataset in a lower-dimensional embedding.
+Importantly, it provides the advantage that the linear mappings from the embedded space can be non-linearised through the use of Gaussian Processes.
+
+Let's start by loading some dependencies.
+```julia
+using Turing
+using AbstractGPs, KernelFunctions, Random, Plots
+
+using Distributions, LinearAlgebra
+using VegaLite, DataFrames, StatsPlots, StatsBase
```

Comment:
Is there a specific reason for using both StatsPlots and VegaLite? Seems a bit complicated to use two different plotting frameworks in one tutorial.